src/gopher.cc

```diff
@@ -102,6 +102,10 @@ class GopherStateData
 
     ~GopherStateData();
 
+    /// URL for icon to display (or nil), given the Gopher item-type code.
```

Comment:
To clarify the second part of my old [change request](https://github.com/squid-cache/squid/pull/1026#discussion_r857135338): We should not document how the URL will be used (e.g. "to display") -- that usage is up to the caller:

```suggestion
    /// Icon URL for the given Gopher item-type code (or nil).
```

This documentation suffers from the "foo is foo" syndrome a bit, but that is the best I can come up with because the method name is already self-documenting in this regard. The other details in the description compensate for this (small) flaw.